Add Notification That Guest Search Will Only Support in B1.0 (#22025)
microsoftgraph/microsoft-graph-docs: concepts/whats-new-overview.md

### Search | Query
- Optionally specify the sortable or refinable properties to collapse in the results of a [search request](/graph/api/resources/searchrequest).
- To include multiple search requests in a single request body in the request body, [use the Microsoft Graph Search API to pass multiple search requests](/graph/search-concept-multiplerequest). 

> - Guest users can search for items within SharePoint or OneDrive that have been shared with them. Guest search will only be available in v1.0.

Reviewer: ```suggestion
- Guest users can search for items within SharePoint or OneDrive that have been shared with them. Guest search is only available in v1.0.
```
